src/frontEnd/volunteering/VolRoutes.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+import { SearchBar as VolSearchBar } from "./SearchBar";
+import { LoadVolunteeringOption } from "./LoadVolunteeringOption";
+import { Router } from "../nav/Router";
+import { regexMatch } from "../../utils/strings/regexMatch";
+import {ViewVolunteeringOption} from "./ViewVolunteeringOption";
+
+
+export const VolRoutes = (url, props, router:Router) => 
+    router
+        .case("/volunteering/search", <VolSearchBar {...props} />)
+        .matches(regexMatch("/volunteering/.+"), <ViewVolunteeringOption {...props} />)
```

Comment:
Maybe extract the matching id out here ideally rather than assume the length of the url in the other part of the code, what do u think?

Reply:
we will do this at a later time